.gitignore

```diff
@@ -3,6 +3,7 @@
 /deps
 erl_crash.dump
 *.ez
+mix.lock
```

Comment:
I just read some of the discussions on this. Thanks for the pointer. (I should have guessed that it was needed by the fact that nobody ignored it yet - duh!)